xml/art_sle_amazon_ecs.xml

```diff
@@ -132,6 +138,69 @@
           cluster. Remember to select the ECS role when launching the
           instance.
         </para>
+        <para>
+          You may use the Amazon EC2 Web Console to launch instances.
+          The SUSE Linux Enterprise Server ECS images are found in the
+          so called General Catalog. From the
+          <emphasis role="strong">Instances</emphasis> view select
+          <emphasis role="strong">Launche Instance</emphasis>. In the left
+          column select the <emphasis role="strong">Community AMIs</emphasis>
+          tab. Check the <emphasis role="strong">SUSE Linux</emphasis> box
+          on the left and enter <emphasis role="strong">ecs</emphasis> into
+          the search box. When configuring the instance parameters in the
+          <emphasis role="strong">Configure Instance Details</emphasis> view
+          select the <emphasis role="strong">ecsInstanceRole</emphasis> as
+          the IAM role. Then launch the instance.
+        </para>
+        <para>
+          Alternatively you can use the aws command line tool to
+          <link xlink:href="http://docs.aws.amazon.com/cli/latest/reference/ec2/run-instances.html">run</link>
+          instances.
+        </para>
+        <screen>$ aws ec2 run-instances --image-id IMAGE_ID_SEE_PINT_OUTPUT --region EC2_REGION_NAME --key-name SSH_KEY_PAIR_NAME --instance-type DESIRED_INSTANCE_TYPE --subnet-id SUBNET_ID --iam-instance-profile 'Arn=PROFILE_PATH'</screen>
+        <para>
+          The PROFILE_PATH may be obtained by querying the IAM service.
```

Comment:
Is PROFILE_PATH a placeholder? A variable name?

* for placeholders, use `<replaceable>`
* for a variable name, use `<varname>`